src/common/modules/navigations/gnb/modules/gnb-search/modules/GNBSearchDropdown.vue

```diff
@@ -99,6 +125,23 @@ export default defineComponent<Props>({
     },
     setup(props, { emit }) {
         const state = reactive({
+            menuSuggestionItems: computed<SuggestionItem[]>(() => {
+                const results: SuggestionItem[] = [];
+                if (props.menuItems.length) {
+                    results.push({ name: 'title', label: props.isRecent ? i18n.t('COMMON.GNB.SEARCH.RECENT_MENU') : i18n.t('COMMON.GNB.SEARCH.MENU'), type: 'header' });
+                    results.push(...props.menuItems);
+                }
+                return results;
```

Comment:
앗 여기 제가 한 부분인 것 같은데

```suggestion
                let results: SuggestionItem[] = [];
                if (props.menuItems.length) {
                    results.push({ name: 'title', label: props.isRecent ? i18n.t('COMMON.GNB.SEARCH.RECENT_MENU') : i18n.t('COMMON.GNB.SEARCH.MENU'), type: 'header' });
                    results = results.concat(props.menuItems);
                }
                return results;
```

요런식으로 concat 으로 변경 부탁드립니다..ㅎ